src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -143,17 +152,24 @@ export class SearchResults extends Component {
 
     return (
       <>
-        <h2 className="vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal">
-          Displaying {resultsStartNumber}-{resultsEndNumber} of {totalResults}{' '}
-          results
+        <h2
+          className="va-introtext vads-u-outline--none vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal"
+          data-forms-focus
```

Comment:
I used `data-forms-focus` just because it was part of Find Forms, but it could be whatever you want (`data-yr-focus`, etc)

Reply:
Oh wow hahaha I totally just copied + pasted this from find-va-forms 😜